vowpalwabbit/simple_label.h

```diff
@@ -11,8 +11,14 @@ struct vw;
 struct label_data
 {
   float label = 0.f;
-  float weight = 0.f;
-  float initial = 0.f;
+  // only used for serialization and parsing.  example.weight is used for
+  // computation
+  // DeSerialized/Parsed values are copied into example in VW::setup_example()
+  float serialized_weight = 0.f;
+  // Only used for serialization and parsing.  example.initial is used for
+  // computation
+  // DeSerialized/Parsed values are copied into example in VW::setup_example()
+  float serialized_initial = 0.f;
```

Comment:
Rather than creating a post-parse step to handle cache files the cache file should just be updated to include this value or we should decide it does not need to be cached.